### ISSUE TITLE
[FEATURE] Safe on Server

### ISSUE THREAD

**Issue by DenuxPlays** [drawdb-io/drawdb#288]
**Is your feature request related to a problem? Please describe.**

My problem is that I cannot start the work on any my devices and seamlessly continue the work on another.

**Describe the solution you'd like**

A way to safe my progress on the server.

**Additional context**

Partially related to: https://github.com/drawdb-io/drawdb/issues/58

Also instead of having a fully user feature you can just create an "account" that will provide you with a sync token that can be used to access your saves.

**Comment by 1ilit** [comment]
Yep there's no seamless way. For now the best way would be to share the diagram everytime you want to switch and use that link. The issue with the server if outlined [here](https://github.com/drawdb-io/drawdb/issues/58#issuecomment-2450878217). Idk if I'm missing smth

**Comment by DenuxPlays** [comment]
Maybe it can be opt-in.
This way you don't have to put up with the whole server things and people that may want this need to self-host the server.
Or the app can link to a server the user wants (enter domain and your sync token).
